### PR TITLE
New scenarios for DOS, more refactoring and updating of G-Cloud SSP

### PR DESCRIPTION
**Pull request by andrewchong**

### CODE REVIEW

**Comment by TheDoubleK** [comment]
Running this locally against preview some things don't work, e.g:
screen shot 2015-12-02 at 16 41 15

Also the curly quotes thing:
screen shot 2015-12-02 at 16 43 02

And also this one for updating framework status:
screen shot 2015-12-02 at 16 44 57

File: features/step_definitions/ssp_steps.rb [hunk 1]
```diff
@@ -99,8 +120,8 @@
 
 Then /^I am taken to the '(.*)' page$/ do |page_name|
   find('h1').should have_content(/#{page_name}/i)
-  store.framework_name = URI.parse(current_url).path.split('frameworks/').last.split('/').first
-  store.service_type = URI.parse(current_url).path.split('submissions/').last.split('/').first
+  #store.framework_name = URI.parse(current_url).path.split('frameworks/').last.split('/').first
```
Comment (TheDoubleK) [inline features/step_definitions/ssp_steps.rb]: It would be good to delete these commented out lines unless we think we might need them again soon.

**Comment by TheDoubleK** [comment]
The step and scenario changes look good.  I also had an odd failure for the removing locations from Agile coach, but tried the steps out manually on Preview and it worked as expected, so I'm not sure if it was a glitch with the app or if it's a bug in the test (the scenario looks fine though):
screen shot 2015-12-02 at 17 28 06

It's all such a moving target at the moment, I reckon get these passing and merge them as soon as you can after that - give me a shout to do a quick review once this is updated so they pass and I'll do it right away.

**Comment by andrewchong** [comment]
Closing this pull request as things have changed significantly that this is no longer valid. A new pull request will be submitted.

**Pull request closed without merging**